R/shiny.R

```diff
@@ -410,7 +410,8 @@ sessionDataURL = function(session, data, id, filter) {
       list(error = as.character(e))
     })
 
-    jsonArgs = c(list(x = res, dataframe = 'rows'), getOption('htmlwidgets.TOJSON_ARGS'))
+    jsonArgs = c(list(x = res, dataframe = 'rows'),
+                 getOption('DT.TOJSON_ARGS', getOption('htmlwidgets.TOJSON_ARGS')))
```

Comment:
Sorry, I'm confused. `getOption('DT.TOJSON_ARGS', getOption('htmlwidgets.TOJSON_ARGS')))` should mean if `DT.TOJSON_ARGS` is NULL, it use the value from `htmlwidgets.TOJSON_ARGS`. Otherwise, it ignores `htmlwidgets.TOJSON_ARGS`. Doesn't it?

`DT.TOJSON_ARGS` | `htmlwidgets.TOJSON_ARGS` | VALUE
---------------------|------------------------------|---------
DT | NULL | DT
DT| htmlwidgets | DT
NULL | htmlwidgets | NULL
NULL | NULL | NULL

Reply:
Sorry I was wrong. You are absolutely correct.